tools/Environments/DevHome.Environments/Views/LandingPage.xaml

```diff
@@ -173,42 +173,6 @@
                 </Grid>
             </DataTemplate>
 
-            <!-- Per Provider Card Template -->
-            <DataTemplate x:Key="PerProviderComputeSystemTemplate" x:DataType="vm:PerProviderViewModel">

```

Comment:
PerProviderComputeSystemTemplate -> ComputeSystemTemplate
These changes are all part of the "Grouping" revert.